compiler/src/utils/warnings.re

```diff
@@ -108,7 +112,9 @@ let message =
   | FuncWasmUnsafe(func) =>
     "it looks like you are using "
     ++ func
-    ++ " on two unsafe Wasm values here.\nThis is generally unsafe and will cause errors. Use one of the equivalent functions in `WasmI32`, `WasmI64`, `WasmF32`, or `WasmF64` instead.";
+    ++ " on two unsafe Wasm values here.\nThis is generally unsafe and will cause errors. Use one of the equivalent functions in `WasmI32`, `WasmI64`, `WasmF32`, or `WasmF64` instead."
+  | FromNumberLiteral32 => "it looks like you are calling Int32.fromNumber() with a constant number. Try using the literal syntax (e.g. `16l`) instead."
+  | FromNumberLiteral64 => "it looks like you are calling Int64.fromNumber() with a constant number. Try using the literal syntax (e.g. `16L`) instead.";
```

Comment:
It'd be really cool if the error message had the actual number they were trying to write with the literal syntax so they could just copy and paste it! What do you think about extending `FromNumberLiteral32` and `64` to have the literal string in it to use that in the message?